example-config.yml

```diff
@@ -369,6 +369,27 @@ itinerary:
 #       - WALK
 #       - BICYCLE
 
+### Popup config
+### OTP-RR supports a full-screen momdal popup that can be launched from a set of
+### pre-set locations. Each button can be configured to open a different popup.
+### Popup "targets" define a URL that is opened. Their ID is used to give them names
+### via the language section below. For example, a target with the id "testTarget"
+### can be named with the i18n string "popups.testTarget"
```

Comment:
Add an example of popup i18n in the `language` section below.

Reply:
Change the nomenclature to `config.popups.testTarget`, so it is consistent with other localizable text that is defined in config.yml.

Reply:
That's a very good idea thanks!